cl/corpus_importer/management/commands/recap_into_opinions.py

```diff
@@ -0,0 +1,124 @@
+from asgiref.sync import async_to_sync
+from django.core.management import BaseCommand
+from django.core.paginator import Paginator
+from django.db import transaction
+
+from cl.lib.command_utils import logger
+from cl.lib.microservice_utils import microservice
+from cl.search.models import (
+    SOURCES,
+    Court,
+    Docket,
+    Opinion,
+    OpinionCluster,
+    RECAPDocument,
+)
+
+
+def import_opinions_from_recap(court=None, total_count=0):
+    """Import recap documents into opinion db
+
+    :param court: Court ID if any
+    :param total_count: The number of new opinions to add
+    :return: None
+    """
+    count = 0
+    if not court:
+        courts = Court.objects.filter(jurisdiction=Court.FEDERAL_DISTRICT)
+    else:
+        courts = Court.objects.filter(id=court)
+    for court in courts:
+        logger.info(f"Importing RECAP documents for {court}")
+        cluster = (
+            OpinionCluster.objects.filter(docket__court=court)
+            .order_by("-date_filed")
+            .first()
+        )
+        if not cluster:
+            print("No clusters for this court")
+            logger.info(f"Court {court} has no opinion clusters")
+
+            continue
+        documents = RECAPDocument.objects.filter(
+            docket_entry__docket__court=court,
+            docket_entry__docket__date_filed__lte=cluster.date_filed,
```

Comment:
Do you mean `gte` or even `gt`? I think you're getting the latest cluster in the court up above, and then you want to get RECAP docs that are newer than that, right? 

And if you do `gte` instead of `gt`, do you run the risk of duplicates on the first date?

Reply:
yes - sorry this was a holdover from testing